### ISSUE TITLE
Not getting an `institution_not_found` metadata status (Native iOS)

### ISSUE THREAD

**Issue by noahemmet** [plaid/link#190]
| Name              | Value              |
| ----------------- | ------------------ |
| Link `apiVersion` |   v2                 |
| Link `env`        |      sandbox              |
| LinkKit version   |    1.0.7                |
| iOS version       | 10.3.1                   |
| iOS device        |   Simulator                 |

### A detailed description of the steps to reproduce the issue

- Set up the configuration ala
  ```swift
  PLKConfiguration(key: "<MY_KEY>", env: .sandbox, product: .auth, selectAccount: true, longtailAuth: false, apiVersion: .PLKAPIv2)
  ```
- Enter the Plaid link flow. 
- Search for a non-existent institution such that the "empty" screen appears.
- Tap the "exit" button.
- Look at the `metadata` object in the `didExitWithError` `PLKPlaidLinkViewDelegate` callback.

### What you expected to see?

- Expected to find a `status: institution_not_found` object in the `metadata`, similar to [what the web view receives](https://plaid.com/docs/api/#onexit-callback).

### What you saw instead?

```
[ 
 "status": <null>, 
 "request_id": <null>, 
 "plaid_api_request_id": <null>
]
```

**Comment by afh** [comment]
Hi @noahemmet,

thank you filing this issue; a fix will be available with the next release of LinkKit.

**Comment by afh** [comment]
Hi @noahemmet,
once #196 is merged and LinkKit 1.0.8 released the issue you encountered is fixed.
